src/picai_prep/mha2nnunet.py

```diff
@@ -109,6 +111,7 @@ def __init__(
         scan_postprocess_func: Optional[Callable[[sitk.Image], sitk.Image]] = None,
         out_dir_scans: PathLike = "imagesTr",
         out_dir_annot: PathLike = "labelsTr",
+        num_threads: int = 4,

```

Comment:
https://stackoverflow.com/a/66799516/6527748
Why 4?

Reply:
This is based on experience. A high number of threads can cause memory issues, as images can be big (and this number of threads is applied throughout). Four threads typically fill up the IO bandwidth (which is the goal of multiprocessing here, which is likely the bottleneck).